### ISSUE TITLE
Content review from Patient Check In

### ISSUE THREAD

**Issue by benbrasso-agile6** [department-of-veterans-affairs/va.gov-team#63061]
## About your team

- Team name: Modernized Check-in Experience
-	OCTO-DE product owner: Kay Lawyer
-	Product manager: Lori Pusey 
-	Designer: Ben Brasso, Zach Park, 
- Accessibility specialist:
- Content writer: Kelly Smith
-	Slack channel: #check-in-experience-ux
 
### Describe what you need in a few sentences:

We have two features that need content review: 

1. Review options for [changing the travel reimbursement page](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/p/697DF8C0-2DF6-4A66-B875-E0594FD8930F/canvas) to eliminate potential concerns ([outlined in this Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1690814320736/04f9a4290906730ed48ca137b9da3c73ecf8c811?sender=u37bb983bd3fc3cc00c7d3286)) from Veterans about filing a travel claim in general, not just specific to check in. Adding comments on an artboard would be best for us. Please tag @benbrasso-agile6 in the comment.
2. Review the [warning alerts content](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/p/1D358457-C022-4ABA-9A38-44171F548862/canvas) we've been working on within the upcoming releases for our unified check in epic for when a Veteran selects a link from a text message and there's an issue. (Here's the [unified check in larger user flow](https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/p/0EC89917-F949-4461-A7B3-32A5201FD2A2/canvas) for more context, if needed, about these scenarios and when they happen.) Adding comments on an artboard would be best for us. Please tag @benbrasso-agile6 in the comment.
</br>

### Which are you doing?
- [ ] **Launching a new** page, tool, form, or app
- [x] **Making changes** to an existing page, tool, form, or app

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [x] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] Object-Oriented UX (OOUX) support
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next step is** .... is to finalize the content in these specific areas. 
- **Provide date if available**: xx/xx/xxxx

#### Timing for your next step: 
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [x] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users on:** Difficult to estimate due to incremental releases and dependencies on other teams and feature requests. 
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: https://app.zenhub.com/workspaces/check-in-experience-61fc23a2cb8a14001132e102/issues/gh/department-of-veterans-affairs/va.gov-team/53488
- **Ticket (number)**: 
- **Collab. Cycle Ticket Opened (date)**: 02/10/2023
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [x] Design Intent
- [x] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **Product outline**:
- **User flows**: https://www.sketch.com/s/0e890de3-2530-4ee0-986e-cf0314334aec/p/0F9F62F0-68A0-4C8B-9105-A92D0A6448DB/canvas#Version
- **Prototype or mockups**:
- **If this is in Sketch, which version or specific page should we focus on?**
- **Content source of truth**:
- **Research plan:** https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1690814320736/04f9a4290906730ed48ca137b9da3c73ecf8c811?sender=u37bb983bd3fc3cc00c7d3286
- **Any epics/issues that may be helpful**:
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [ ] Danielle
- [ ] Beth
- [ ] Mikki
- [ ] Martha

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by cgednalske** [comment]
@laurwill I've added the H1 question @kellysmith1008 and I are working on ([Slack thread](https://dsva.slack.com/archives/C02GXKL8WM6/p1691181915722709))  to this list so everything is in one place. Let us know if you have any questions or want to talk through anything.
cc @benbrasso-agile6

**Comment by laurwill** [comment]
Thanks @cgednalske and @benbrasso-agile6 ! Great to have this all tracked in one ticket — we'll work this into our queue for content review.

**Comment by laurwill** [comment]
Hi @kellysmith1008 @cgednalske , I replied to some comments in the updated sketch files — is the H1 question tracking somewhere else now? I don't see it in the list anymore. Thanks!

**Comment by kellysmith1008** [comment]
Hi @strelichl ! I _think_ we're good, but want to make sure @benbrasso-agile6 signs off on it as well.

**Comment by benbrasso-agile6** [comment]
Good to close, TY
